client/StatusTable/components/StatusTable.js

```diff
@@ -0,0 +1,66 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import {connect} from 'react-redux'
+import * as actions from '../actions'
+import WeekRow from './WeekRow'
+
+class StatusTable extends Component {
+    componentDidMount() {
+        this.props.tableLoadInit();
+    }
+
+    componentDidUpdate(prevProps) {
+        if (this.props.startSelected !== prevProps.startSelected) {
+            this.props.tableLoadInit();
+        }
+    }
+
+    // before this we lose _id from parent component
+    // componentWillUnmount() {
+    //     const { _id } = this.props;
+    //     this.props.tableCloseInit({_id});
+    // }
+
+    render() {
+        const { workweeks }= this.props;
+        if (!workweeks || !workweeks.length) {
+            // when TABLE_LOAD_FAILURE
+            return (null);
+        }
+
+
+        const rows = workweeks.map(workweek => <WeekRow workweek={workweek} startSelected={this.props.startSelected} key={workweek.numberOfTheWeek} />);
+
+        return (
+            <React.Fragment>
```

Comment:
no need in fragment there as you already render only one component. use Fragment when there is more than one